decentraland/unity-explorer
# Fix: multiplayer movement uncompressed with rotation

## What does this PR change?

- brought back uncompressed message
- add rotation for uncompressed message
- updated respective proto to include rotation for uncompressed message
- add CLI parameter to use compression
- exposed several settings parameters to debug panel

## How to test the changes?

It is a multiplayer feature, so playtest with 2 players - one on Win, another on Mac

1. Launch the explorer
2. Verify your friend avatar replicated and not floating away after some time

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
